spring-pulsar/src/main/java/org/springframework/pulsar/core/PulsarOperations.java

```diff
@@ -38,7 +38,7 @@
 	 * @throws PulsarClientException if an error occurs
 	 */
 	default MessageId send(T message) throws PulsarClientException {
-		return send(null, message);
+		return send(null, message, null);
```

Comment:
I made all of the default impls call to the abstract one rather than daisy-chan into the next most specific `send` method. This prevents coupling between the different APIs (ie `send(T message)` only depends on the abstract `send(null, message, null);` whereas before it depended on `send(null, message);` which then depended on `send(null, message, null);` which then would mean that we have to consider all 3 when changing one of them. Basically 2 dependent function calls is less coupling than 3.